pyproject.toml

```diff
@@ -36,8 +36,12 @@ pythonpath = [
   "src"
 ]
 markers = [
-  "mount_snap: mark a test as using /SNS/SNAP/ data mount"
+  "integration: mark a test as an integration test",
+  "mount_snap: mark a test as using /SNS/SNAP/ data mount",
+  "golden_data(*, path=None, short_name=None, date=None): mark golden data to use with a test"
 ]
+addopts = "-m 'not integration'"
```

Comment:
spelling error?

Reply:
I'm not sure what you mean.  The "addopts" line works for me.  Can you clarify?